policies/lib/attestations.rego

```diff
@@ -19,3 +19,24 @@ taskrun_attestations := result {
 		att.predicate.buildType == taskrun_att_build_type
 	]
 }
+
+hacbs_test_task_result_name := "HACBS_TEST_OUTPUT"
+
+# (Don't call it test_results since test_ means a unit test)
+results_from_tests := [r |
+	att := pipelinerun_attestations[_]
+	task := att.predicate.buildConfig.tasks[_]
```

Comment:
Would it work to create a `tasks` variable similar to `pipelinerun_attestations`?

Reply:
Seems to work. https://github.com/hacbs-contract/ec-policies/pull/37/commits/ede9f00af2e62d1541bc5c84e4f9b0d4538b038a

Reply:
Nice, I added this in latest revision. Also realized that pipelinerun_attestations could be just a var not a rule.